src/main/java/build/buildfarm/tools/WorkerProfile.java

```diff
@@ -68,10 +68,11 @@ private static void analyzeMessage(String worker, WorkerProfileMessage response)
     System.out.printf(
         (strIntFormat) + "%n", "Current Unreferenced Entry Count", unreferencedEntryCount);
     if (entryCount != 0) {
-      System.out.printf(
-          (strFloatFormat) + "%n",
-          "Percentage of Unreferenced Entry",
-          1.0 * response.getCasEntryCount() / response.getCasUnreferencedEntryCount());
+      System.out.println(
+          format(
```

Comment:
I think this line needs to be removed

Reply:
I've pushed the changes, however @werkt already included these changes along with additional  fixups in https://github.com/buildfarm/buildfarm/pull/2054